client/client_test.go

```diff
@@ -283,7 +323,7 @@ func TestGetSTH(t *testing.T) {
 
 			gotSTH, gotHTTPData, gotErr := lc.GetSTH()
 			if gotErrType := reflect.TypeOf(gotErr); gotErrType != test.wantErrType {
-				t.Errorf("GetSTH(): error was of type %v, want %v", gotErrType, test.wantErrType)
+				t.Fatalf("GetSTH(): error was of type %v, want %v", gotErrType, test.wantErrType)
```

Comment:
This was intentionally not a Fatal call - the error types not matching may not be the only thing wrong with what's returned, as the HTTPData returned always contains at least timing info. It'd be useful to still perform the next few checks and have all of those errors reported together instead of having to re-run the tests.

Reply:
Oops, fixed.